ouj/python/algorithm/binary_search.py

```
@@ -0,0 +1,21 @@
+        if query == target[center]:
+            return True
+        elif query < target[center]:
+            end = center - 1
+        else:
+            start = center + 1
+    return False
+
+
+n = int(input("n?"))
+query = int(input("query?"))
+binary_search([i for i in range(n)], query)
```

Ruff (0.16.2)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)